queue_entry.py

```diff
@@ -26,7 +26,6 @@
 import time
 import queue_model_objects_v1 as queue_model_objects
 import os
-import ShapeHistory as shape_history
```

Comment:
shape_history is in use in line 556:
`pos = shape_history.Point(None, cpos, None) #, True)`
So it has to be kept no ?